packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -274,7 +274,7 @@ export const BootDataProvider = ({
         isFetched={isBootReady}
         isLegacyLogout={bootData?.isLegacyLogout}
         accessToken={bootData?.accessToken}
-        isPastRegistration={isInitialFetch.current}
+        isPastRegistration={isInitialFetch}
```

Comment:
Can you check auth flows if they would still work as intended.
- Register using email
- Register using email but refresh the page on account details form.
- Register using provider
- Register using provider but refresh the page on account details form.